README.org

```diff
@@ -56,6 +56,7 @@ In addition to the provider, which you may want multiple of (for example, to cha
 - ~:port~: The port that ollama is run on.  This is optional and will default to the default ollama port.
 - ~:chat-model~: The model name to use for chat.  This is not optional for chat use, since there is no default.
 - ~:embedding-model~: The model name to use for embeddings.  This is not optional for embedding use, since there is no default.
+- ~:endpoint~: The ollama endpoint to use, either "generate" or "chat".  This is optional and will default to "generate".
```

Comment:
Do you think it makes a difference to use "generate", when we could just use "chat" always?  If there's no appreciable difference, I'd prefer not having an option.